provisioner/template.go

```diff
@@ -675,3 +676,20 @@ func certificateExpiry(certificate string) (string, error) {
 	}
 	return expiry.UTC().Format(time.RFC3339), nil
 }
+
+func subQuantity(quantities ...string) (string, error) {
+	var result k8sresource.Quantity
```

Comment:
It looks like (did not check) that Quantity supports sign https://pkg.go.dev/k8s.io/apimachinery/pkg/api/resource#Quantity

If so it might be better to introduce `sumQuantities` to support both addition and subtraction.
`subQuantity("1", "100m", "1m")` would be then `sumQuantities("1", "-100m", "-1m")`

Reply:
I changed it to `subQuantities`. It makes the templating on kubernetes-on-aws side a little more verbose, but seems reasonable when it's more generic.

Reply:
Changed to `sumQuantities` I guess.